Automattic/jetpack
# Jetpack Setup Wizard: Add setup wizard intro page

#### Changes proposed in this Pull Request:

This PR starts the new Jetpack Setup Wizard by adding the intro page.

Note that the buttons and links do not work at the time; they will be updated to work as their respective pages are added.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2

Figma design:
<img width="953" alt="image" src="https://user-images.githubusercontent.com/42627630/79584839-8ef29880-8094-11ea-87bd-505662bb7d39.png">

Figma design mobile:
<img width="282" alt="image" src="https://user-images.githubusercontent.com/42627630/79585196-10e2c180-8095-11ea-8a2a-6a8cb74a8b74.png">

Current state:
<img width="1679" alt="image" src="https://user-images.githubusercontent.com/42627630/79584896-a3cf2c00-8094-11ea-9197-7a81cdfd0ba4.png">

Current state mobile:
<img width="351" alt="image" src="https://user-images.githubusercontent.com/42627630/79585383-5606f380-8095-11ea-9e03-0dca1b7fae52.png">


#### Testing instructions:





1. Add the following to the end of wp-config.php: `add_filter( 'jetpack_show_setup_wizard', '__return_true' );`
1. Visit the Jetpack plugin. Verify that the setup wizard intro page is the first page you see.
1. Resize the page and make sure the responsive behavior looks good.
1. Verify that the masthead, footer, etc do not show on the intro page.
1. Toggle between the set up, dashboard, settings, and site stats pages. Verify that the appropriate sub-navigation item in the left sidebar is highlighted as being selected.
1. In the Jetpack dashboard, verify that the masthead, footer, apps card, support card, and rewind status still show.
1. In the Jetpack settings page, verify that the masthead and footer still show.
1. Remove the `jetpack_show_setup_wizard` filter from wp-config.php and reload the page.
1. Toggle between the dashboard and settings pages. Verify that the appropriate sub-navigation item in the left sidebar is highlighted as being selected.
1. Visit the Jetpack plugin. Verify that the dashboard is the first page you see. 
1. Visit  `/wp-admin/admin.php?page=jetpack#/setup`. Verify that you are redirected to the dashboard.

#### Proposed changelog entry for your changes:
* Added new setup wizard
